HW1/HW1/Sublines.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace HW1
+{
+    public class Sublines
+    {
+        private string line;
+
+        public Sublines(string line)
+        {
+            this.line = line;
+        }
+        
+
+        public void PrintSublines()
+        {
+            line.ToLower();
+            Console.WriteLine("Line: " + line);
+            Console.WriteLine("Sublines: ");
+            for (int index = 0; index < line.Length; index++)
+            {
```

Comment:
Better to add all substrings to collection, and then write to console in one cycle